pkgs/applications/office/super-productivity/default.nix

```diff
@@ -1,15 +1,15 @@
-{ stdenv , lib , fetchurl , appimageTools , makeWrapper , electron_11 }:
+{ stdenv , lib , fetchurl , appimageTools , makeWrapper , electron_12 }:
 
 let
-  electron = electron_11;
+  electron = electron_12;
```

Comment:
Would be better to move it to top level:
```
  super-productivity = callPackage ../applications/office/super-productivity {
    electron = electron_12;
  };

```

Reply:
Done :-)